install_config/install/prerequisites.adoc

```diff
@@ -127,6 +127,9 @@ above, for each 1000 pods. Therefore, the recommended size of master host in an
 well as the minimum requirements for a master host of 2 CPU cores and 8 GB of
 RAM.
 
+When planning a multi-master environment, a minimum of three etcd hosts as well
```

Comment:
I'm not sure on the short version "multi" here. I suggest:

When planning an environment with multiple masters...